force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +444,9 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+            if(person1.getAge() >= person2.getAge()){
+                return person1;
+            }
+        return person2; // Replace null with the variable you used to store the result
```

Comment:
Although it's not needed, it's good to add the else statement for easier readability.